spec/client_spec.rb

```diff
@@ -209,7 +209,7 @@
       end
 
       context 'with non-serializable data' do
-        let(:data) { [:foo, 'bar'] }
```

Comment:
We still wants symbols to fail — can we keep the corresponding test?

Reply:
I might be slow, it's still January. :)

Are keys, or keys and values converted from symbols to strings?

Reply:
Just keys, values still raise an error:

```
➜ ~ irb
irb(main):001:0> require 'oj'
=> true
irb(main):002:0> Oj.default_options = {:mode => :strict}
=> {:mode=>:strict}
irb(main):003:0> Oj.dump({'key' => :value})
TypeError: Failed to dump Symbol Object to JSON in strict mode.
```